### PR TITLE
JetBrains: add ability to edit account from context menu

### PR DESCRIPTION
**Pull request by szymonprz**
It's possible now to edit account from context menu (right click on account) 
Screenshot 2023-09-11 at 10 14 35

## Test plan
- Right click on existing account and edit some fields, account should be updated after clicking "Apply" on dialog, reset and closing the dialog without applying the settings should not change account
- Add and remove should still work

### CODE REVIEW

File: client/jetbrains/src/main/kotlin/com/sourcegraph/cody/config/AddCodyAccountWithTokenAction.kt [hunk 1]
```diff
@@ -4,41 +4,40 @@ import com.intellij.openapi.actionSystem.AnActionEvent
 import com.intellij.openapi.actionSystem.PlatformCoreDataKeys
 import com.intellij.openapi.project.DumbAwareAction
 import com.intellij.openapi.project.Project
-import com.intellij.openapi.util.NlsContexts
 import com.sourcegraph.cody.api.SourcegraphApiRequestExecutor
 import java.awt.Component
 import javax.swing.JComponent
 
 class AddCodyAccountWithTokenAction : BaseAddAccountWithTokenAction() {
-    override val defaultServer: String
```
Comment (olafurpg) [inline client/jetbrains/src/main/kotlin/com/sourcegraph/cody/config/AddCodyAccountWithTokenAction.kt:13]: Is there a reason that ktfmt didn't catch these whitespaces changes before?

**Review by olafurpg — APPROVED** [review]
I just tried this locally and it works great! Thank you.

File: client/jetbrains/src/main/kotlin/com/sourcegraph/cody/config/AddCodyAccountWithTokenAction.kt (hunk 1 above)
Comment (szymonprz) [reply client/jetbrains/src/main/kotlin/com/sourcegraph/cody/config/AddCodyAccountWithTokenAction.kt:13]: I have no idea why it changed and I don't want to waste time to investigate that :/

Comment (olafurpg) [reply client/jetbrains/src/main/kotlin/com/sourcegraph/cody/config/AddCodyAccountWithTokenAction.kt:13]: No worries

**Pull request merged**